uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/controllers/leave-request-popup-controller.js

```diff
@@ -266,6 +270,15 @@ define([
       };
 
       /**
+       * Checks if popup is opened in view mode
+       *
+       * @return {Boolean}
+       */
+      vm.isView = function () {
```

Comment:
it's a good idea but isView was created for the UI to have less comparison code, i think in JS the comparison are manageable